admin/sql/sitemaps/CreateFKConstraints.sql

```diff
@@ -1,7 +1,7 @@
 -- Automatically generated, do not edit.
 \set ON_ERROR_STOP 1
 
-SET search_path = 'sitemaps';
+SET search_path = 'sitemaps', 'musicbrainz', 'public';
```

Comment:
This is necessary because the `earthdistance` extension expects its function names (in the `public` schema) to be always visible. (This can probably be considered a bug: http://www.postgresql.org/message-id/flat/CALy1bpd2rDKk-V246KD79NTSBGYP0j7L4N+yQ33U85uRUUevoQ@mail.gmail.com) Due to the way foreign key constraints are added internally in Postgres, the problem would be triggered below when a FK to `place` is defined.